ocaml/idl/datamodel.ml

```diff
@@ -6883,6 +6885,7 @@ let vm =
 	field ~writer_roles:_R_VM_ADMIN ~qualifier:RW ~in_product_since:rel_boston ~default_value:(Some (VRef (Ref.string_of Ref.null))) ~ty:(Ref _sr) "suspend_SR" "The SR on which a suspend image is stored";
 	field ~qualifier:StaticRO ~in_product_since:rel_boston ~default_value:(Some (VInt 0L)) ~ty:Int "version" "The number of times this VM has been recovered";
 	field ~qualifier:StaticRO ~in_product_since:rel_clearwater ~default_value:(Some (VString "0:0")) ~ty:(String) "generation_id" "Generation ID of the VM";
+	field ~writer_roles:_R_POOL_OP ~qualifier:StaticRO ~in_product_since:rel_creedence ~default_value:(Some (VInt 0L)) ~ty:Int "virt_hw_vn" "The host virtual hardware version the VM can running on";
```

Comment:
This should be _R_VM_ADMIN: the person administrating the VM should be allowed to mark it as able to run on a particular virtual hardware platform version. (If this turns out to be wrong then the VM will fail to run, but this will not be a problem for the people who only care about the hosts or pools.)